### ISSUE TITLE
PSR4 Loader bug

### ISSUE THREAD

**Issue by alex7r** [joomla/joomla-cms#17413]
### Steps to reproduce the issue
Place class `App\Application` at `joomlaroot/App/Application.php`;
Register Namespace `JLoader::registerNamespace('App',JPATH_SITE.DIRECTORY_SEPARATOR.'App',false,false,'psr4');`
Call for `App\Application`

### Expected result
Class loaded

### Actual result
Class not loaded

### Additional comments
At psr4 loader:
```
‌‌$classPath
‌App/Application.php

‌‌$classFilePath
‌/vagrant/html/App//lication.php
```

**Comment by laoneo** [comment]
Please test #17470, it should fix the issue. Thanks for reporting.

**Comment by ghost** [comment]
closed as having Pull Request #17470<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17413">issues.joomla.org/tracker/joomla-cms/17413</a>.</sub>
